catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/filestorage/MetacardFileStorageRoute.java

```diff
@@ -96,6 +109,11 @@ public void refresh(Map<String, Object> properties) throws Exception {
     super.refresh(properties);
   }
 
+  @Override
+  public List<String> getRouteIds() {
```

Comment:
Does this result in these routes showing up when executing camel:route-list in the karaf console?

Reply:
These routes do show up when executing `camel:route-list`